src/magic_folder/uploader.py

```diff
@@ -230,19 +232,21 @@ def add_file(self, path):
             self._status.upload_queued(self._config.name, relpath)
         except ValueError:
             ADD_FILE_FAILURE.log(relpath=path.path) #FIXME relpath
-            raise ValueError(
-                "The path being added '{!r}' is not within '{!r}'".format(
+            raise APIError(
```

Comment:
Same comment per spreading Web code .. a non-Web-specific error should be raised here.